notebooks/metrics_ex_02.ipynb

```diff
@@ -102,6 +102,15 @@
     "compute the $R^2$ score and the mean absolute error for instance."
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Write your code here."
+   ]
+  },
```

Comment:
Hum, do we have a regression in the notebook script?

```suggestion
```

Or was the deduplication of `# Write your code here.` cells done manually in the past?

If so, we should definitely follow-up with another PR to improve our script to automatically trim those repeated cells.

Reply:
There is no regression. https://github.com/INRIA/scikit-learn-mooc/pull/788 was never merged.